exchange/exchange.go

```diff
@@ -139,6 +139,7 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 			}
 		}
 	}
+	targData.cacheHost, targData.cachePath = e.cache.GetPrebidCacheSplitURL()
```

Comment:
We only need cacheHost and cachePath if we are supplying other targData, so it can go into the if above and you don't need to precreate a targData struct

Reply:
You are right Hans, this was a bug. Corrected